buildtools/npm/manifest_test.go

```diff
@@ -1,65 +1,111 @@
 package npm_test
 
 import (
-	"errors"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
 	"github.com/fossas/fossa-cli/buildtools/npm"
+	"github.com/fossas/fossa-cli/pkg"
 )
 
+func getDirectDepImports() (chaiDirectDep pkg.Import, typeDetectDirectDep pkg.Import) {
+	chaiDirectDep = pkg.Import{
+		Target: "chai",
+		Resolved: pkg.ID{
+			Location: "",
+			Name:     "chai",
+			Revision: "4.1.2",
+			Type:     pkg.NodeJS,
+		},
+	}
+
+	typeDetectDirectDep = pkg.Import{
+		Target: "type-detect",
+		Resolved: pkg.ID{
+			Location: "",
+			Name:     "type-detect",
+			Revision: "3.0.0",
+			Type:     pkg.NodeJS,
+		},
+	}
+
+	return
+}
+
 func TestFromManifest(t *testing.T) {
-	manifest, err := npm.FromManifest("testdata/package.json")
+	pkg, err := npm.PackageFromManifest(filepath.Join("testdata", "nested_node_modules/"), "package.json")
```

Comment:
`nested_node_modules/` should not have the trailing slash. This defeats the purpose of using `filepath.Join`.